packages/legacy-client/src/client/baseTypes/ActionType.ts

```diff
@@ -141,8 +163,8 @@ export const ActionResponse = {
       ],
     };
     if (response.edits?.type === "edits") {
-      const added = [];
-      const modified = [];
+      const added: any[] = [];
```

Comment:
Didn't touch this, but for some reason had to add this type, or else checks error out with something along the lines of :
`Argument of type '{ apiName: string; primaryKey: any; get: () => Promise<Result<OntologyObject<string, NonNullable<ParameterValue>>, GetObjectError>>; }' is not assignable to parameter of type 'never'.ts(2345)`

Reply:
Hmm. I am a little worried this means we broke something somewhere. If we need to keep a change like this we could at least type it to the real shape we are pushing to prevent future errors.